### PR TITLE
[optin/puller]: add batch scripts

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: optin/puller/scripts/setup.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PYTHONPATH=. python3 workflows/download_postoptin.py \
+	--nem-node "${NEM_NODE}" \
+	--symbol-node "${SYMBOL_NODE}" \
+	--database-directory "${DATABASE_DIRECTORY}" \
+	--optin-address "${OPTIN_ADDRESS}" \
+	--snapshot-height "${SNAPSHOT_HEIGHT}"
+PYTHONPATH=. python3 workflows/get_account_states.py \
+	--node "${NEM_NODE}" \
+	--database-directory "${DATABASE_DIRECTORY}" \
+	--snapshot-height "${SNAPSHOT_HEIGHT}"
```
Comment (gimre-xymcity) [inline optin/puller/scripts/setup.sh:22]: not sure if this should be here or rather in run

File: optin/puller/scripts/testnet.env [hunk 2]
```diff
@@ -0,0 +1,11 @@
+NETWORK_NAME=testnet
+DATABASE_DIRECTORY=_temp/TESTNET
+SNAPSHOT_HEIGHT=3471104
+
+PREOPTIN_JSON=resources/empty_preoptin.json
+OPTIN_ADDRESS=TCTYYOMGVB6IDGQBKVZOCDDV6MQMM6YZBBK2AT2Z
```
Comment (gimre-xymcity) [inline optin/puller/scripts/testnet.env:6]: `optin_addresses` ;)

Comment (Jaguar0625) [reply optin/puller/scripts/testnet.env:6]: hm? seems all address parameters are singular

Comment (gimre-xymcity) [reply optin/puller/scripts/testnet.env:6]: but this arg allows multiple addresses, right? (coma separated)

**Review by gimre-xymcity — APPROVED** [review]

**Pull request merged**